diktat-analysis.yml

```
@@ -0,0 +1,230 @@
+- name: DIKTAT_COMMON
+  enabled: true
+  configuration:
+    isCopyrightMandatory: true
+    domainName: please enter your domain name
+    copyrightText: please enter your copyright text
```

```suggestion
    isCopyrightMandatory: false
    domainName: org.cqfn.diktat
    copyrightText:
```
The config from root directory will be used for diktat code check